NumberPlateScheme.csv

```diff
@@ -0,0 +1,100 @@
+﻿Start,End,Plate Id

```

Comment:
Ultimately this file wants to live in `inst` or `inst/data` http://r-pkgs.had.co.nz/data.html#data-data